gensim/models/keyedvectors.py

```diff
@@ -189,7 +189,28 @@
 logger = logging.getLogger(__name__)
 
 
-KEY_TYPES = (str, int, np.integer)
+_KEY_TYPES = (str, int, np.integer)
+
+_EXTENDED_KEY_TYPES = (str, int, np.integer, np.ndarray)
+
+
+def _ensure_list(value):
+    #
```

Comment:
Why not a standard `""""` docstring, for these two lines? (the rationale below can stay as comment)

Sphinx doesn't render docs for `_underscore_methods`, right?

Reply:
No particular reason. I'll make it into a docstring.

Yes, this docstring won't end up in the docs, because the function is _internal.